.github/workflows/ci.yml

```diff
@@ -373,15 +373,42 @@ jobs:
         with:
           key: clap
 
-      - name: Run semver-checks
+      - name: Run semver-checks with --baseline-root
         continue-on-error: true
         run: |
           cargo run semver-checks check-release --manifest-path="subject-current/Cargo.toml" --baseline-root="subject-baseline/Cargo.toml" > output
-          echo "$?" > return_code
+          touch failed_successfully
```

Comment:
Perhaps `unexpectedly_did_not_fail` might be a better name? `failed_successfully` sounds like the opposite of what this file's existence signifies, no?

Reply:
Done in ace4bbcd024fa889d8df91fbd3a7f147606e05a5.